<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="btnServe">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>430</y>
         <width>189</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Serve Token</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGenerate">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>430</y>
         <width>201</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>GenerateToken</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExit">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>430</y>
         <width>191</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>-10</y>
         <width>611</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Token Management System</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>661</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lblTokenNumber">
          <property name="text">
           <string>Token Number :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblCustomerName">
          <property name="text">
           <string>Customer Name:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblPurpose">
          <property name="text">
           <string>Purpose:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblGeneratedAt">
          <property name="text">
           <string>Generated At:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblServedAt">
          <property name="text">
           <string>Served At:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lblStatus">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLineEdit" name="editName">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>80</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter customer name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editPurpose">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>120</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter Purpose</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
